lib/reaction-wheel/reaction-wheel.cpp

```diff
@@ -0,0 +1,141 @@
+#include <Arduino.h>
```

Comment:
Consider moving this file to the include directory. That is where most of our functions are.

Reply:
I'd like to keep the reaction wheel elements more isolated to allow for vertical extensibility cause we're working on the same program across two teams, if there is further need we can merge more closely in the future

Reply:
From the initial design spec user functions are in the include directory

Reply:
We can still revert to a previous working version and we are currently working together on the same board and flight software so why not?

Reply:
it's more about maintainability and separation of concerns, we could discuss a revised spec or improvements to the current spec in the future